mivs/site_sections/mivs_applications.py

```diff
@@ -0,0 +1,151 @@
+from mivs import *
```

Comment:
This is how we make new HTML pages.  Every function defined here is a page handler, e.g. the ``studio`` function corresponds to the url at ``/uber/mivs_applications/studio``.  The return value of each function is a dictionary which contains the data which will be used to render the templates, e.g. the ``studio`` function will render the template at ``mivs/templates/mivs_applications/studio.html``.